WeakAurasOptions/OptionsFrames/Update.lua

```diff
@@ -1213,6 +1213,50 @@ local function ListAuras(auras, uidMap)
   return result
 end
 
+local function AddAuraList(container, uidMap, list, expandText)
+  local expand = AceGUI:Create("WeakAurasExpand")
+  local collapsed = true
+  local image = collapsed and "Interface\\AddOns\\WeakAuras\\Media\\Textures\\expand"
+                           or "Interface\\AddOns\\WeakAuras\\Media\\Textures\\collapse"
+  expand:SetImage(image)
+  expand:SetImageSize(10, 10)
+  expand:SetFontObject(GameFontHighlight)
+  expand:SetFullWidth(true)
+  expand:SetLabel(expandText)
+  container:AddChild(expand)
+
+  local auraLabelContainer = AceGUI:Create("WeakAurasInlineGroup")
+  auraLabelContainer:SetFullWidth(true)
+  auraLabelContainer:DoLayout()
+  container:AddChild(auraLabelContainer)
+
+  local sortedNames = {}
+  for uid in pairs(list) do
+    tinsert(sortedNames, uidMap:GetIdFor(uid))
+  end
+  table.sort(sortedNames)
+
+  expand:SetCallback("OnClick", function()
+    collapsed = not collapsed
+    local image = collapsed and "Interface\\AddOns\\WeakAuras\\Media\\Textures\\expand"
+                           or "Interface\\AddOns\\WeakAuras\\Media\\Textures\\collapse"
+    expand:SetImage(image)
+
+    if collapsed then
+      auraLabelContainer:ReleaseChildren()
+    else
+      for _, name in ipairs(sortedNames) do
+        local auraLabel = AceGUI:Create("Label")
+        auraLabel:SetText("   • " .. name)
+        auraLabel:SetFullWidth(true)
+        auraLabelContainer:AddChild(auraLabel)
+      end
```

Comment:
```suggestion
      local text
      for _, name in ipairs(sortedNames) do
        text = (text or "") .. "   • " .. name .. "\n"
      end
      if text then
        local auraLabel = AceGUI:Create("Label")
        auraLabel:SetText(text)
        auraLabel:SetFullWidth(true)
        auraLabelContainer:AddChild(auraLabel)
      end
```
This have same result but is much faster with huge list like 
![image](https://user-images.githubusercontent.com/189656/168343277-4de5431e-7251-4da8-9dcd-c340f7b1fee3.png)